core/src/main/java/org/apache/shiro/authc/pam/ModularRealmAuthenticator.java

```diff
@@ -100,7 +100,7 @@ public ModularRealmAuthenticator() {
     ============================================*/
 
     /**
-     * Sets all realms used by this Authenticator, providing PAM (Pluggable Authentication Module) configuration.
+     * Set al. realms used by this Authenticator, providing PAM (Pluggable Authentication Module) configuration.
```

Comment:
Thats a false positive

Reply:
Technically not a false positive, just a really careless replacement -- oops. Will drop